plugins/plugin-svn/che-plugin-svn-ext-ide/src/main/java/org/eclipse/che/plugin/svn/ide/commit/CommitViewImpl.java

```diff
@@ -0,0 +1,360 @@
+        changesElement.setAttribute("style", "width: 100%; background: none;");
+        changesElement.setCellSpacing("1px");
+        changesList = SimpleList.create((SimpleList.View)changesElement, coreRes.defaultSimpleListCss(), new ChangesListRenderer(),
+                                        listChangesDelegate);
+
+        changesWrapper.add(changesList);
+
+        message.getElement().setAttribute(PLACEHOLDER, locale.commitPlaceholder());
+
+        alertMarker = resources.alert().getSvg();
+        alertMarker.getStyle().setWidth(22, Style.Unit.PX);
+        alertMarker.getStyle().setHeight(22, Style.Unit.PX);
+        alertMarker.getStyle().setMargin(10, Style.Unit.PX);
+        getFooter().getElement().appendChild(alertMarker.getElement());
+
+        Tooltip.create((elemental.dom.Element)alertMarker.getElement(),
+                       PositionController.VerticalAlign.TOP,
+                       PositionController.HorizontalAlign.MIDDLE,
+                       locale.commitMessageEmpty());
+
+        alertMarker.getStyle().setVisibility(Style.Visibility.VISIBLE);
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public void onClose() {
+        hide();
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public void setDelegate(final ActionDelegate delegate) {
+        this.delegate = delegate;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public void onShow() {
+        commitAll.setValue(true);
+        commitSelection.setValue(false);
+        show();
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public String getMessage() {
+        return this.message.getText();
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public boolean isKeepLocksStateSelected() {
+        return keepLocks.getValue();
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public boolean isCommitAllSelected() {
+        return commitAll.getValue();
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public boolean isCommitSelectionSelected() {
+        return commitSelection.getValue();
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public void setChangesList(List<StatusItem> changes) {
+        changesList.render(changes);
+        changedFilesCount.setText(String.valueOf(changes.size()));
+    }
+
+    @UiHandler("message")
+    @SuppressWarnings("unused")
+    public void onMessageChanged(KeyUpEvent event) {
+        btnCommit.setEnabled(!message.getText().isEmpty());
+
+        alertMarker.getStyle().setVisibility(!message.getText().isEmpty() ? Style.Visibility.HIDDEN : Style.Visibility.VISIBLE);
+    }
+
+    @UiHandler({"commitAll", "commitSelection"})
+    @SuppressWarnings("unused")
+    public void onCommitModeChanged(ClickEvent event) {
+        delegate.onCommitModeChanged();
+    }
+
+    private class ChangesListRenderer extends SimpleList.ListItemRenderer<StatusItem> {
+
+        class FileStatusColor {
+            private static final String ADDED      = "#629755";
```

Comment:
Not sure. This PR does not aim to solve all problems in plugin-svn. There is an [epic](https://jira.codenvycorp.com/browse/CHE-255) that references all related issues that will need to be solved once the plugin is available on master.
I added an issue to check colors for both themes.